market/connector/microsoft365/toDo/meta.json

```diff
@@ -13,16 +12,23 @@
 	"mavenArtifacts": [
 		{
 			"repoUrl": "https://maven.axonivy.com",
-			"name": "Microsoft Graph ToDo Demo",
+			"name": "Microsoft Graph Product",
 			"groupId": "com.axonivy.connector.office365",
-			"artifactId": "msgraph-todo-demo"
+			"artifactId": "msgraph-connector-product",
```

Comment:
haven't we spent so much time to craft 4 distinct products? ... shouldn't we then also consume unique products in each of these meta.json files?

Reply:
2 comments up you said I should iclude the ZIP. 
should I not do it then?